app/components/participation-list.hbs

```diff
@@ -38,19 +38,20 @@
   </ul>
 </div>
 {{/if}}
-{{#if (not @readOnly)}}
-<AuButton
-  @width="block"
-  @skin="secondary"
-  @icon="pencil"
-  @iconAlignment="left"
-  {{on 'click' this.togglePopup}}
-  @disabled={{this.fetchMandatees.isRunning}}
-  class="au-u-margin-top-small"
->
-  {{@modalTitle}}
-</AuButton>
-{{/if}}
+
+{{#unless @readOnly}}
```

Comment:
I'm really not a fan of this. I think the `{{#if (not something)}}` combo is way more readable, but the linting rule doesn't have an option to allow it.